corehq/apps/geospatial/management/commands/index_utils.py

```diff
@@ -4,29 +4,29 @@
 from corehq.apps.geospatial.es import case_query_for_missing_geopoint_val
 from corehq.form_processor.models import CommCareCase
 from corehq.util.log import with_progress_bar
+from corehq.apps.geospatial.const import DEFAULT_CHUNK_SIZE, DEFAULT_QUERY_LIMIT
 
 
 def process_batch(
-    domain, geo_case_property, case_type,
-    query_limit,
-    chunk_size,
+    domain,
+    geo_case_property,
     total_count,
     with_progress=False,
     offset=0,
 ):
-    should_sort = bool(total_count > query_limit)
+    should_sort = bool(total_count > DEFAULT_QUERY_LIMIT)
     query = case_query_for_missing_geopoint_val(
-        domain, geo_case_property, case_type, size=query_limit, offset=offset, should_sort=should_sort
+        domain, geo_case_property, case_type, size=DEFAULT_QUERY_LIMIT, offset=offset, should_sort=should_sort
```

Comment:
I see that `size` was removed above in this commit for this function?

Reply:
That is correct, I'm simply using `DEFAULT_QUERY_LIMIT` for size in the query. This was the default value being passed around by both the task and the management command, so I thought it makes more sense to simply just use this default instead of passing a param to `case_query_for_missing_geopoint_val`

Reply:
Sorry, I meant to point that size was not a permitted kwarg anymore.